reference/import_export/product-import.rst

```diff
@@ -1,68 +1,81 @@
-Understand the CSV Product Import
-=================================
+Understand the Product Import
+=============================
 
-We'll discuss here how the csv product import works.
+We'll discuss here how the product imports works.
 
 It's a good start to understand the overall architecture and how to re-use or replace some parts.
 
 .. note::
 
-  The import part has been widely re-worked in 1.4. Since 1.6, the old import system is deleted; the following documentation is related to this new system.
+  The import part has been widely re-worked in 1.6. Since 1.6, the old import system is deleted; the following documentation is related to this new system.
```

Comment:
The import [...] in 1.6. Old import system has been removed, please refer to previous versions of this page if needed.